docs/reference/config.rst

```diff
@@ -716,6 +716,39 @@ Default: ``{}`` (empty).
 MusicBrainz Options
 -------------------
 
+Default configuration::
```

Comment:
Just for my own sanity, could we keep this PR's docs changes focused on the new option? Some refactoring, reordering, and adding examples would be nice too, but it will be easier to think about in a separate PR. Thank you!

Reply:
Of course, I'll move them out to another branch!